docs/source/index.rst

```diff
@@ -11,6 +11,9 @@ Dialog Flow Framework
    :alt: Number of downloads
    :target: https://pypi.org/project/dff/
 
+.. image:: https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-green.svg
+   :alt: Supported python versions
+
 The Dialog Flow Framework (DFF) is an open-source, `Apache 2.0 <https://www.apache.org/licenses/LICENSE-2.0>`_-licensed library
 that was developed specifically for creating dialog systems. DFF provides a comprehensive set of tools and resources for building conversational systems
 for a wide range of applications, including chatbots, virtual assistants, and other interactive systems.
```

Comment:
```suggestion
that was developed specifically for creating dialog systems. DFF provides a comprehensive set of tools and resources, targeting a wide range of applications, including chatbots, virtual assistants, and other interactive systems.
```
dialog systems == conversational systems

Reply:
Not that related to PR, but I changed that too